backend/src/v5/middleware/dataConverter/inputs/teamspaces/projects/models/metadata.js

```diff
@@ -17,37 +17,46 @@
 
 const { createResponseCode, templates } = require('../../../../../../utils/responseCodes');
 const Yup = require('yup');
-const { getMetadataById } = require('../../../../../../models/metadata');
+const { getMetadataByQuery } = require('../../../../../../models/metadata');
 const { respond } = require('../../../../../../utils/responder');
 
 const Metadata = {};
 
 const metadataSchema = Yup.object().shape({
-	key: Yup.string().min(1).max(120).required(),	
+	key: Yup.string().min(1).max(120).required(),
```

Comment:
Can't we just use `title` (https://github.com/3drepo/3drepo.io/blob/staging/backend/src/v5/utils/helper/yup.js#L43) here? am I missing something?